.storybook/preview.ts

```diff
@@ -44,6 +44,10 @@ export const parameters = {
     breakpointNames,
     debounceTimeout: 10,
   },
+  docs: {
+    toc: true,
```

Comment:
For now I'd prefer this config:

```suggestion
    toc: {
      ignoreSelector: '.docs-story h2',
      headingSelector: 'h2',
    },
```